module/wa/module.py

```diff
@@ -967,18 +968,44 @@ def import_wa(i):
             pname=None
             imported_params={}
 
-            for name, obj in inspect.getmembers(cs):
-                if name!='AndroidUxPerfWorkload' and inspect.isclass(obj):
-                    try:
-                        addr=getattr(cs, name)
-                        pp=addr.parameters
-                        pname=addr.package
-                    except Exception as e:
-                        pass
-
-                    if pp!=None:
-                        wa_name=name
-                        break
+            # Only examine classes
+            for name, obj in inspect.getmembers(cs, predicate=inspect.isclass):
```

Comment:
Originally I didn't want to change more code than I needed to, so adopted the existing mechanism for loading/checking classes. Using ExtensionLoader seems like an improvement - but requires some other changes in the import code which is currently running 'wa list workloads' as a command to maintain consistency and then looks like it needs a little refactoring to make the most of the changes.

This might not fit with the overall project ethos, which I think would like to avoid programmatic dependencies on the services it wraps, but that would be up to @gfursin or @psyhtest I think.

I'd rather work those changes up as an additional patch, as the current load_module approach is already in place.